ngx-fudis/projects/ngx-fudis/src/lib/utilities/storybook.ts

```diff
@@ -439,23 +442,30 @@ export const spacingExclude: RegExp = excludeRegex([
   'handleFocus',
   'icon',
   'iconRotate',
+  'id',
   'label',
   'labelHidden',
+  'popoverPosition',
+  'popoverText',
+  'popoverTriggerLabel',
   'size',
-  'type',
-  'variant',
   'toggleMenu',
   'tooltip',
   'tooltipPosition',
   'tooltipToggle',
+  'type',
+  'variant',
 ]);
 
 /**
  * Text Input
  */
 export const textInputExclude: RegExp = excludeRegex([...formCommonDocsExclude]);
 
-export const textInputControlsExclude: RegExp = excludeRegex([...formCommonControlsExclude]);
+export const textInputControlsExclude: RegExp = excludeRegex([
+  ...formCommonControlsExclude,
+  'name',
```

Comment:
Text-input `name` property actually should be visible since it is a required (or highly advised) input when using the `autocomplete` input (I think it is a good idea to enable this feature in the controls as well). Previously it wasn't working due to not existing in the story arguments.

Best to initialize `name` with empty string

```
const commonArgs: Partial<TextInputComponent> = {
...
  autocomplete: 'off',
  name: '',
};
```

and add it to the template

```
  [autocomplete]="autocomplete"
  [name]="name"
```